crates/common/mqtt_channel/src/tests.rs

```diff
@@ -403,8 +403,15 @@ mod tests {
             .publish(topic, "A 2nd msg published before clean")
             .await?;
 
-        // If we clean the session
-        clear_session(&mqtt_config).await?;
+        // Then we clean the session
+        {
+            // One just needs a config with the same session name.
+            // Subscriptions can be given - but this not required: any previous subscriptions will be cleared.
+            let mqtt_config = Config::default()
```

Comment:
Why not re-use the mqtt_config from before? (if its to test that different mqtt_config can both clear a session, maybe both cases should be tested?)

Reply:
The point is to highlight that only the session name matters.